engine/src/main/java/io/camunda/zeebe/engine/state/signal/DbSignalSubscriptionState.java

```diff
@@ -43,15 +43,15 @@ public DbSignalSubscriptionState(
     signalNameAndSubscriptionKey = new DbCompositeKey<>(signalName, subscriptionKey);
     signalNameAndSubscriptionKeyColumnFamily =
         zeebeDb.createColumnFamily(
-            ZbColumnFamilies.SIGNAL_SUBSCRIPTION_BY_NAME_AND_KEY,
+            ZbColumnFamilies.DEPRECATED_SIGNAL_SUBSCRIPTION_BY_NAME_AND_KEY,
```

Comment:
🤔 Does this cause any other issues except warnings?

Reply:
Well. This is only needed because we copy the latest version of the ZbColumnFamilies enum, where this column family has been renamed. So we have to update references. The only problem I see is accidentally forgetting an update right now.

This is why I'm not so eager to copy the entire column families enum, actually.

@Zelldon What do you think? Is it worth the risk? Personally, I'd like to remove 44366d6

Reply:
More and more I feel I don't want to copy the entire column families enum from main. It introduces another chance of using the wrong column families, but in a different way. Simply because the production code is using these column families and we may miss or wrongly update them now when copying the enum from main.

@Zelldon [Here's an example](https://github.com/camunda/zeebe/pull/16169/commits/08e6c63e6bd184367e9ae1fae83b655cf7c98ed5) of what it looks like to not copy the entire file but just fix the issue on 8.1.

The changes are easier to review now, without risking usage of wrong column families.

Really curious to your thoughts

Reply:
@Zelldon agreed with me when we discussed this on slack. 

So, I'm removing 44366d6 from this PR. Copying the `ZbColumnFamilies` enum from `main` is too error prone.